### PR TITLE
No Data animation showing in case of empty data from API response

### PR DESCRIPTION
**Pull request by AbdulRehmanNazar**
No Data animation showing when there is empty data coming from API response.

https://github.com/alirezaeiii/TMDb-Compose-Playground/assets/6792823/8c674afe-03ed-4f8c-bd08-025d606c9f37

### CODE REVIEW

File: buildSrc/src/main/java/Dependencies.kt [hunk 1]
```diff
@@ -58,8 +59,10 @@ object Deps {
     const val composeUiTooling = "androidx.compose.ui:ui-tooling:${Versions.compose}"
     const val composeUiToolingPreview = "androidx.compose.ui:ui-tooling-preview:${Versions.compose}"
     const val navigationCompose = "androidx.navigation:navigation-compose:${Versions.nav_version}"
-    const val composeConstraintLayout = "androidx.constraintlayout:constraintlayout-compose:${Versions.constraint_layout}"
```
Comment (alirezaeiii) [inline buildSrc/src/main/java/Dependencies.kt:61]: Is there any reason you add a space here?

File: buildSrc/src/main/java/Dependencies.kt [hunk 2]
```diff
@@ -75,8 +78,10 @@ object Deps {
 
     // Core
     const val lifecycleLiveData = "androidx.lifecycle:lifecycle-livedata-ktx:${Versions.lifecycle}"
-    const val lifecycleViewModel = "androidx.lifecycle:lifecycle-viewmodel-ktx:${Versions.lifecycle}"
```
Comment (alirezaeiii) [inline buildSrc/src/main/java/Dependencies.kt:78]: And here also.

File: features/feature-paging/src/main/java/com/sample/tmdb/paging/search/SearchScreens.kt [hunk 3]
```diff
@@ -85,6 +86,9 @@ fun <T : TMDbItem> Search(
     @StringRes resourceId: Int,
     modifier: Modifier = Modifier
 ) {
+
```
Comment (alirezaeiii) [inline features/feature-paging/src/main/java/com/sample/tmdb/paging/search/SearchScreens.kt]: And space here.

File: features/feature-paging/src/main/java/com/sample/tmdb/paging/search/SearchScreens.kt [hunk 4]
```diff
@@ -98,6 +102,7 @@ fun <T : TMDbItem> Search(
             ) {
                 AnimatedSearch()
             }
+
```
Comment (alirezaeiii) [inline features/feature-paging/src/main/java/com/sample/tmdb/paging/search/SearchScreens.kt]: Space here

**Review by alirezaeiii — CHANGES_REQUESTED** [review]
Your idea is very interesting. Thank you for your contribution. If you have any other idea, please feel free to open a PR.

File: buildSrc/src/main/java/Dependencies.kt [hunk 5]
```diff
@@ -58,8 +59,10 @@ object Deps {
     const val composeUiTooling = "androidx.compose.ui:ui-tooling:${Versions.compose}"
     const val composeUiToolingPreview = "androidx.compose.ui:ui-tooling-preview:${Versions.compose}"
     const val navigationCompose = "androidx.navigation:navigation-compose:${Versions.nav_version}"
-    const val composeConstraintLayout = "androidx.constraintlayout:constraintlayout-compose:${Versions.constraint_layout}"
-    const val iconExtended = "androidx.compose.material:material-icons-extended:${Versions.icon_extended}"
+    const val composeConstraintLayout =
```
Comment (alirezaeiii) [inline buildSrc/src/main/java/Dependencies.kt]: Is there any reason you add a space here?

File: buildSrc/src/main/java/Dependencies.kt [hunk 6]
```diff
@@ -75,8 +78,10 @@ object Deps {
 
     // Core
     const val lifecycleLiveData = "androidx.lifecycle:lifecycle-livedata-ktx:${Versions.lifecycle}"
-    const val lifecycleViewModel = "androidx.lifecycle:lifecycle-viewmodel-ktx:${Versions.lifecycle}"
-    const val lifecycleSavedstate = "androidx.lifecycle:lifecycle-viewmodel-savedstate:${Versions.lifecycle}"
+    const val lifecycleViewModel =
```
Comment (alirezaeiii) [inline buildSrc/src/main/java/Dependencies.kt]: And here.

**Comment by AbdulRehmanNazar** [comment]
@alirezaeiii 
Thanks for your feedback. some of the spaces are auto generated by studio, now its reset.
please go ahead and check and let me know if there is any issue any
Love to solve them

**Review by alirezaeiii — APPROVED** [review]

**Comment by alirezaeiii** [comment]
> @alirezaeiii Thanks for your feedback. some of the spaces are auto generated by studio, not its reset. please go ahead and check and let me know if there is any issue any Love to solve them

Approved, Thanks again, Feel free to share any other new idea.

**Pull request merged**